sdk/spring-sdk/src/it/java/com/example/wiring/SpringSdkWiringIntegrationTest.java

```diff
@@ -109,6 +108,91 @@ public void verifyCounterEventSourceSubscription() {
             new IsEqual(42 + 1));
   }
 
+  @Test
+  public void verifyActionIsNotSubscribedToMultiplyAndRouterIgnores(){
+
+     webClient
+            .post()
+            .uri("counter/counterId2/increase/1")
+            .retrieve()
+            .bodyToMono(Integer.class)
+            .block(timeout);
+    webClient
+            .post()
+            .uri("counter/counterId2/multiply/2")
+            .retrieve()
+            .bodyToMono(Integer.class)
+            .block(timeout);
+
+    webClient
+            .post()
+            .uri("counter/counterId2/increase/1234")
+            .retrieve()
+            .bodyToMono(Integer.class)
+            .block(timeout);
+
+    Integer counterGet =
+            webClient.get().uri("/counter/counterId2").retrieve().bodyToMono(Integer.class).block(timeout);
+
+    //Before has time to process any action things still work
+    Assertions.assertEquals(1*2 + 1234, counterGet);
+
+    //Once it tries to process actions will fail
```

Comment:
but this is not failing and we are calling the entity here

what am I missing?

Reply:
misleading comment. Changed now